docs/cloud/pouta/adding-images.md

```diff
@@ -173,6 +173,128 @@ you need to add the line "*acpiphp*" to */etc/modules*. For other
 distros, please check how to load *acpiphp* on boot from the distro
 documentation.
 
+
+## Converting images
+
+When performing a snapshot of a virtual machine, OpenStack creates an 
+image in _raw_ format. These images typically take as many GB as the 
```

Comment:
This might not always be the default behavior, in our case it's something we have configured explicitly due to the dataloss bug

https://github.com/CSCfi/cccp/blob/master/manifests/role/compute.pp#L78

Reply:
I dunno, it might be fine as is, or could throw in a disclaimer like `Pouta OpenStack is configured to create`

Reply:
Or just 

```
-OpenStack creates an image in _raw_ format.
+the resulting image is created in _raw_ format.
```

Up to you, could also be I'm overthinking a bit :)